File: data/evaluation/geeks_for_geeks_successful_test_scripts/python/COUNT_FREQUENCY_K_MATRIX_SIZE_N_MATRIXI_J_IJ.py
# Copyright (c) 2019-present, Facebook, Inc.
# All rights reserved.
#
# This source code is licensed under the license found in the
# LICENSE file in the root directory of this source tree.
#
def f_gold ( n , k ) :
    if ( n + 1 >= k ) :
        return ( k - 1 )
    else :
        return ( 2 * n + 1 - k )


#TOFILL

if __name__ == '__main__':
    param = [
    (90,74,),
    (86,36,),
    (92,38,),
    (72,71,),
    (25,57,),
    (11,53,),
    (94,80,),
    (91,75,),
    (66,58,),
    (34,88,)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        if f_filled(*parameters_set) == f_gold(*parameters_set):
            n_success+=1
    print("#Results: %i, %i" % (n_success, len(param)))

File: data/evaluation/geeks_for_geeks_successful_test_scripts/cpp/COUNT_FREQUENCY_K_MATRIX_SIZE_N_MATRIXI_J_IJ.cpp
// Copyright (c) 2019-present, Facebook, Inc.
// All rights reserved.
//
// This source code is licensed under the license found in the
// LICENSE file in the root directory of this source tree.
//

#include <iostream>
#include <cstdlib>
#include <string>
#include <vector>
#include <fstream>
#include <iomanip>
#include <bits/stdc++.h>
using namespace std;
int f_gold ( int n, int k ) {
  if ( n + 1 >= k ) return ( k - 1 );
  else return ( 2 * n + 1 - k );
}


//TOFILL

int main() {
    int n_success = 0;
    vector<int> param0 {90,86,92,72,25,11,94,91,66,34};
    vector<int> param1 {74,36,38,71,57,53,80,75,58,88};
    for(int i = 0; i < param0.size(); ++i)
    {
        if(f_filled(param0[i],param1[i]) == f_gold(param0[i],param1[i]))
        {
            n_success+=1;
        }
    }
    cout << "#Results:" << " " << n_success << ", " << param0.size();
    return 0;
}